packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -401,6 +401,15 @@ export const prodParasPolkadotCommon: EndpointOption[] = [
       Pinknode: 'wss://public-rpc.pinknode.io/statemint',
       RadiumBlock: 'wss://statemint.public.curie.radiumblock.xyz/ws'
     }
+  },
+  {
+    info: 'polkadotCollectives',
+    paraId: 1001,
+    text: 'Collectives',
+    teleport: [-1],
```

Comment:
Collectives does not support teleportation (or reserve backed transfers) yet. We need https://github.com/paritytech/polkadot/pull/6326 first.